### PR TITLE
JW-880 log4cplus

### PR DESCRIPTION
**Pull request by mclapinski**
Again, it's a lot of code 'cause I restored some removed code.

### CODE REVIEW

File: lib/windowsstubs/SConscript [hunk 1]
```diff
@@ -1,46 +1,19 @@
...
 abs_vpath = Dir(vpath).abspath
-abs_toppath = Dir('#').abspath
-
-x64dir = ''
-if env['TARGET_ARCH'] == 'x64':
-   x64dir = r'\x64'
+windowsstubs_lib = '#build/lib/windowsstubs.lib'
 
-#build libraries
 def build_windowsstubs_lib(target,source,env):
```
Comment (jablonskim) [inline lib/windowsstubs/SConscript]: Maybe add spaces

**Review by jablonskim — APPROVED** [review]

**Comment by michal-clapinski** [comment]
Retest this please

**Comment by mclapinski** [comment]
Retest this please

**Comment by mclapinski** [comment]
Retest this please

File: src/SConscript [hunk 2]
```diff
@@ -44,8 +44,11 @@ if sys.platform != 'darwin':
 include = ['#/controller/src', '#/build/include', '#controller/lib']
 
 libpath = ['#/build/lib']
-#WINDOWS-TEMP
-libs = ['boost_system', 'log4cplus', 'libboost_system-vc140-mt-gd-1_62', 'libboost_date_time-vc140-mt-gd-1_62', 'libboost_regex-vc140-mt-gd-1_62', 'libboost_program_options-vc140-mt-gd-1_62', 'libboost_thread-vc140-mt-gd-1_62', 'libboost_filesystem-vc140-mt-gd-1_62']
+
+if sys.platform.startswith('win'):
```
Comment (Attil) [inline src/SConscript:48]: Sometimes you decide the platform via `if platform.system() == 'Windows'` and sometimes `if sys.platform.startswith('win')`. Please decide on one.

**Review by Attil — APPROVED** [review]

**Pull request merged**